index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Create Mobile Version</title>
+</head>
+<body>
+    <main>
+        
+        <header>
+            <h1 class="logo">Welcome</h1>
```

Comment:
- [ ] You should use `<a>` tag instead of `<h1>` because this is a logo and it is good practice to make the logo clickable to take the user to the home page.

Reply:
<a> tag has been used in place of <h1> on the <header> as suggested.